markdown/epilogue.markdown

```diff
@@ -1,39 +1,151 @@
 # Epilogue
 
+> Quality means doing it right when no one is looking.
+>
+> — Henry Ford
+
+Our main goal with this book was promoting elegant uses of the NumPy and SciPy
+libraries, and, we hope, inspiring in you the feeling that quality code is
+something worth striving for, while teaching you how to do effective scientific
+analysis with SciPy.
```

Comment:
Suggest:

Our main goal with this book was to promote elegant uses of the NumPy and SciPy libraries.
While teaching you how to do effective scientific analysis with SciPy we hope that we've inspired in you the feeling that quality code is something worth striving for.